gridmet_aggregation_PRMS.py

```diff
@@ -0,0 +1,105 @@
+# libraries
+import geopandas as gpd
```

Comment:
I realized when I ran this that we need `os` b/c of line 22
```suggestion
import os
import geopandas as gpd
```